src/theme/components/customTable.ts

```diff
@@ -105,17 +111,17 @@ export const baseCellStyles = (
   lineHeight: 1.5,
   paddingBottom: isScrollable ? "s" : { base: "0", md: "s" },
   paddingEnd: "s",
-  paddingStart: columnHeadersBackgroundColor ? { base: "0", md: "s" } : "s",
+  paddingStart: isScrollable ? "s" : { base: "0", md: "s" },
   paddingTop: isScrollable ? "s" : { base: "0", md: "s" },
   _first: {
     borderBottom: showRowDividers
-      ? isScrollable
-        ? "1px solid var(--nypl-colors-ui-gray-semi-medium)"
-        : {
-            base: "1px solid var(--nypl-colors-ui-gray-semi-medium)",
-            md: "1px solid var(--nypl-colors-ui-gray-semi-medium)",
-          }
+      ? "1px solid var(--nypl-colors-ui-gray-semi-medium)"
       : "none",
+    _dark: {
+      borderBottom: showRowDividers
+        ? "1px solid var(--nypl-colors-ui-gray-x-dark)"
+        : "none",
```

Comment:
Is there a meaningful difference between undefined and none?

Reply:
`"none"` is equivalent to `"0"`. Using `undefined` and `null` causes the attribute to not be rendered. That being said, while `undefined` and `null` do essentially the same thing, we have seen differing results with each, but off the top of my head I can not remember the exact differences.